client/consensus/babe/src/lib.rs

```diff
@@ -486,6 +556,11 @@ impl<B: BlockT> BabeWorker<B> {
 		self.slot_notification_sinks.lock().push(sink);
 		stream
 	}
+
+	/// Get a handle to the worker.
+	pub fn handle(&self) -> BabeWorkerHandle<B> {
+		self.handle.clone()
```

Comment:
This handle is for sending `BabeRequest` right? but looks like you don't expose the inner `Sender` from `BabeWorkerHandle`, unless this is still a WIP.